### ISSUE TITLE
Debian package depends on xserver-xorg-input-void, removed in bullseye

### ISSUE THREAD

**Issue by stefanor** [jitsi/jibri#423]
## Description
The jibri debian package depends on `xserver-xorg-input-void`. This was [removed from Debian](https://bugs.debian.org/955603) in bullseye.
Not sure if it is strictly needed?

## Current behavior
The `jibri` Debian package fails to install on bullseye, due to a dependency error.

## Expected Behavior
The `jibri` Debian package should install.

## Possible Solution
Drop the dependency on `xserver-xorg-input-void` if not necessary?

## Steps to reproduce
Spin up a Debian bullseye VM / container, add the https://download.jitsi.org APT repo (`deb https://download.jitsi.org stable/`), and attempt to `apt install jibri`

# Environment details
Debian 11 (bullseye)

**Comment by felbinger** [comment]
I've got the same issue, @stefanor did you find a solution to fix the problem yourself?

**Comment by emrahcom** [comment]
It seems that `xserver-xorg-input-void`  is only needed for `xorg-xserver` which is older than `version 1.4`. The latest `xorg-server-1.13` released on `2013-04-17` (_xorg-server-1.13.4.tar.bz2_).

See [xorg-server releases](https://www.x.org/releases/individual/xserver/)

According to `man void`

```
void is an dummy/null Xorg input driver... It's purpose is to allow X servers
pre version 1.4 to operate without a core pointer and/or core keyboard.
```

`xserver-xorg-core` version according to distributions
* `Ubuntu 16.04 Xenial Xerus` has `xserver-xorg-core 1.18.4`
* `Ubuntu 18.04 Bionic Beaver` has `xserver-xorg-core 1.19.6`
* `Ubuntu 20.04 Focal Fossa` has `xserver-xorg-core 1.20.11`
* `Debian 10 Buster` has `xserver-xorg-core 1.20.4`
* `Debian 11 Bullseye` has `xserver-xorg-core 1.20.11`

I tested `jibri` without `xserver-xorg-input-void` and it seems working.

After the following commands I can record the meeting session without any problems.
```bash
dpkg -P --force-depends xserver-xorg-input-void
systemctl stop jibri-icewm.service
systemctl start jibri.service
```

**Comment by saghul** [comment]
Fixed in https://github.com/jitsi/jibri/pull/440
